execution/block-partitioner/src/tests.rs

```diff
@@ -104,198 +89,6 @@ fn test_relative_ordering_for_sender() {
         });
 }
 
-#[test]
```

Comment:
@sitalkedia deleting this test case because there are some v1/naive v2-specific assertions...

`test_partitioner_v2_*_correctness()` should have the correctness covered.